src/screens/TermsScreen.tsx

```diff
@@ -0,0 +1,150 @@
+import { useRef, useState } from 'react'
+import { Dimensions } from 'react-native'
+import { ScrollView } from 'react-native-gesture-handler'
+import { FontAwesome } from '@expo/vector-icons'
+import * as WebBrowser from 'expo-web-browser';
+import Markdown from 'react-native-markdown-display'
+
+import { Box, Button } from '@components'
+import { palette } from '@utils/theme'
+const dimensions = Dimensions.get('screen')
+
+type TermsScreenProps = {
+  onAccept?: Function,
+  onExit?: Function
+}
+
+export const TermsScreen: React.FunctionComponent<TermsScreenProps> = ({ onAccept, onExit }) => {
+  // Custom link handler
+  const onLinkPress = (url) => {
+    if (url) {
+      // Open in a browser modal
+      WebBrowser.openBrowserAsync(url, {
+        toolbarColor: palette.purple,
+        controlsColor: palette.darkGrey
+      })
+      return false;
+    }
+
+    // return true to open with `Linking.openURL
+    // return false to handle it yourself
+    return true
+  }
+
+  return (
+    <>
+      <ScrollView
+        scrollEventThrottle={16}
+        contentInsetAdjustmentBehavior="automatic"
+        style={{
+          position: 'absolute',
+          height: '100%',
+        }}
+      >
+        {/* Terms and Condition Text */}
+        <Box paddingHorizontal={4} paddingTop={10} paddingBottom={10}>
```

Comment:
Do you prefer long-form over shortcut? `<Box px={4} py={10} />`

Reply:
I didn't know about shorthand....

Reply:
It has its pros and cons. Pros: Less things to read. Less cognitive overhead once you get used to it. Cons: Learning curve for people unfamiliar with it.

I have a loose preference to shorthand just because it's faster to type and faster to read once you're used it.

Reply:
For reference, shorthand keys are in `[]` [here](https://github.com/Shopify/restyle#predefined-restyle-functions).